docs/user_guide/workloads/run_tensorflow.md

```diff
@@ -24,17 +25,69 @@ specific language governing permissions and limitations
 under the License.
 -->
 
-Here is an example for Tensorflow job. You must install tf-operator first. 
-You can install tf-operator by applying all yaml from two website down below:
-* CRD: https://github.com/kubeflow/manifests/tree/master/tf-training/tf-job-crds/base
-* Deployment: https://github.com/kubeflow/manifests/tree/master/tf-training/tf-job-operator/base
-Also you can install kubeflow which can auto install tf-operator for you, URL: https://www.kubeflow.org/docs/started/getting-started/
+This guide gives an overview of how to set up [training-operator](https://github.com/kubeflow/training-operator)
+and how to run a Tensorflow job with YuniKorn scheduler. The training-operator is a unified training operator maintained by
+Kubeflow. It not only supports TensorFlow but also PyTorch, XGboots, etc.
 
-A simple Tensorflow job example:
+## Install training-operator
+You can run one command to install training operator in kubeflow namespace. If you have problems with installation,
+please refer to [this doc](https://github.com/kubeflow/training-operator#installation) for more details.
+```
+kubectl apply -k "github.com/kubeflow/training-operator/manifests/overlays/standalone?ref=v1.3.0"
```

Comment:
missing `-n kubeflow` in this command? since earlier it said to install in kubeflow namespace

Reply:
Thanks @yangwwei 
The manifests is in kubeflow namespace by default, so "-n kubeflow" can be omitted.

ref: https://github.com/kubeflow/training-operator/blob/master/manifests/overlays/standalone/kustomization.yaml#L3

Reply:
Ah, I see